### PR TITLE
Add translation for `and` word on the integration card

### PR DESCRIPTION
**Pull request by bieniu**
## Breaking change

None

## Proposed change

This PR adds translation for `and` world on the integration card.

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by bramkragten — APPROVED** [review]

File: src/translations/en.json [hunk 1]
```diff
@@ -1377,6 +1377,7 @@
             "stop_ignore": "Stop ignoring"
           },
           "config_entry": {
+            "and": "and",
```
Comment (bramkragten) [inline src/translations/en.json:1380]: We might want to move this to generic.

Comment (bieniu) [reply src/translations/en.json:1380]: Move this to `ui.common`?

Comment (bramkragten) [reply src/translations/en.json:1380]: Yeah

Comment (bieniu) [reply src/translations/en.json]: Done

**Pull request merged**